src/components/provider/UALProvider.js

```diff
@@ -112,8 +112,6 @@ export class UALProvider extends Component {
        * @return {Void}
        */
       showModal: () => {
-        const { availableAuthenticators } = this.state
-        availableAuthenticators.forEach(auth => auth.reset())
```

Comment:
What's the rationale for remove this line?

Reply:
`auth.reset()` resets the authenticators to default and then calls the init method again, which adds to the initialization calls.

Reply:
I guess this is fine.  A thing to keep in mind is the edge case where a developer would need their app to explicitly reset an active authenticator before calling ``showModal`` - otherwise the modal won't ever show up.